packages/docs/code-snippets/external/example-image-gallery/src/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Web3.Storage | Image gallery example</title>
+  </head>
+  <body>
+    <header>
+      <a href="https://web3.storage">
+        <img src="https://web3.storage/w3storage-logo.svg" style="height:1.8rem" />
```

Comment:
@yusefnapora hot linking alert... this path may not still exist in the new website. 

Consider hot-linking to external resources by CID / gateway url, they, in theory, last longer. Or better still, be self contained so your root CID contains all the things.